lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -80,11 +85,35 @@ def licences
         )["results"]
       end
 
-      def filtered_licences
-        licence_identifiers = File.new(common_licences_path).readlines(chomp: true)
+      def licences_tagging
```

Comment:
I think this method and `licence_tagging` are too similarly named, and are therefore easy to misread and get confused.

Reply:
Sure, I’ve renamed `licence_tagging`